tests/AbstractTestCase.php

```diff
@@ -32,4 +32,11 @@ protected function assertDqlWhere(?array $expectedDqlWhere, QueryBuilder $queryB
             $this->assertSame($expectedDqlWhere, $where->getParts());
         }
     }
+
+    protected function createBlankCallback(string $name): callable
```

Comment:
It is not blank, it throws exception.

Reply:
its blank - since it should not be called, it is just a callable placeholder

Reply:
I think too that `blank` is not right name.

Reply:
ok.. what would you suggest then? It is the best I've came up with, if you have better idea (and there obviously is some), please tell me - comments like `its it not right` is IMHO useless - if you have a better solution - share it, otherwise say nothing..

Reply:
`createThrowingCallback`
`createExceptionThrowingCallback`
`createErroneusCallback`

But right now i just don`t get what this callback is good for so i can hardly provide some meaningful name.

Reply:
It creates function (dummy callable) - it is exactly what I would do with string like this:
```php
$blankString = "foo";
```
but with functions - there is too much bolier plate around
```
$blankCallable = function() {
    // do nothing
}
```
The meaning of those examples is the same - create _some_ placeholder of concrete type, but not use its value..

Reply:
If you need it just to fulfill dependecies and it meant not to be called, call it `createDummyCallback`. "Dummy" is how things like this should be called  (https://martinfowler.com/bliki/TestDouble.html). "blank" is indeed misleading.